README.txt

```diff
@@ -57,17 +57,21 @@ Installation
 Any platform
 1. Download the latest zip release from https://github.com/openpreserve/fido/releases
 2. Unzip into some directory
-3. Open a command shell, cd to the directory that you placed the zip contents into and cd into folder 'fido'
-4. You should now be able to see the help text:
-   python fido.py -h
-5. Before identifying files with FIDO for the first time, please update signatures first
-   using the 'update_signatures.py' script (see below for instructions).
+3. Open a command shell, cd to the directory that you placed the zip contents into
+4. Run `python setup.py install` to install FIDO and dependencies.  This may require sudo.
+5. You should now be able to see the help text:
+   fido -h
+
+Using pip
+1. Run `sudo pip install opf-fido`
```

Comment:
"It depends", I think. It can work without sudo on some platforms and not others?